### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by iAmWaris97**
**Description**
This React capstone project is about building a mobile web application to check a list of metrics you will create using React and Redux.

**General requirements**

- There are no linter errors.
- I used the correct Gitflow.
- I documented my work.
- HTML/CSS & JavaScript requirements are met.

**Project requirements:**

- Used coin stats API.
- Create react app.
- Used redux.
- Used iconscout.
- Styled the app for mobile first.

### CODE REVIEW

File: src/components/Coins/Coins.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  width: 100%;
+  min-height: 200px;
+  background-color: #5384e1;
+  padding: 25px 0;
+  box-sizing: border-box;
+}
+
+.odd-bg {
+  background-color: #4c7bd3;
+}
```
Comment (omar-muhamad) [inline src/components/Coins/Coins.css]: - [ ] Please make the light and dark background of the coin cards staggered like a chessboard, not like straps as you did to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: src/components/Coins/Coin.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  gap: 30px;
+  align-items: center;
+  padding: 10px 30px;
+  box-sizing: border-box;
+  margin-bottom: 20px;
+}
+
+.info img {
+  border-radius: 50%;
+  height: 150px;
```
Comment (omar-muhamad) [inline src/components/Coins/Coin.css:22]: - [ ] Please use the correct font families: `gill sans` for the body and title and `lato` as control font.

image

File: README.md [hunk 3]
```diff
@@ -1 +1,89 @@
...
+## Author
+### 👨‍💻 Waris Haleem
+- GitHub: [@iAmWaris97](https://github.com/iAmWaris97)
+- Twitter: [@iAmWaris97](https://twitter.com/iAmWaris97)
+- LinkedIn: [Waris Haleem](https://www.linkedin.com/in/waris-haleem/)
+
+## Acknowledgments
+
+> Thanks to all the developers whose code, documentations, and efforts helped us doing this project.
+
```
Comment (omar-muhamad) [inline README.md:77]: - [ ] In README here at the "Acknowledgements", Please give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @iAmWaris97,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by iAmWaris97** [comment]
Hi @Omar-Muhamad,

Thank you for your kind review and comments.
I have made the necessary changes.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @iAmWaris97,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**